utils/templates.py

```diff
@@ -261,3 +261,55 @@ def __init__(self, params=None, content=None):
 
         self.body = copy.deepcopy(self.params)
         self.body.update({'content': self.content})
+
+class ShortReviewReply(InvitationReply):
+    """
```

Comment:
This looks good, but I would like to (very soon) remove the whole /utils directory (the openreview-scripts repo isn't really designed to handle modules like this, and it's causing problems). Could you move this to the conference's config.py file?